typescript/packages/well-log-viewer/src/SyncLogViewer.tsx

```diff
@@ -64,7 +64,7 @@ export interface SyncLogViewerProps {
     /**
      * Object from JSON file describing single well log data.
      */
-    welllogs: WellLog[];
+    welllogs: (WellLogSet | WellLogCollection)[];
```

Comment:
rename to wellLogSets ?

Reply:
As discussed, added a **new** prop named `wellLogSets`